q2_alignment/_mafft.py

```diff
@@ -53,6 +54,11 @@ def mafft(sequences: DNAFASTAFormat,
     result = AlignedDNAFASTAFormat()
     aligned_fp = str(result)
 
+    if not parttree and len(ids) > 1000000:
+        raise ValueError(
+            "The number of sequences in your feature table is larger than "
+            "1 million, please use the parttree parameter")
```

Comment:
I suspect this should be a warning rather than an error. @thermokarst ?

Reply:
No, I think it should be an error - the underlying mafft tool errors, this just replaces their message w a q2-specific message.

Reply:
Hey @turanoo, maybe you should add a comment to the code just above the `ValueError` that explains _why_ we are raising our own error. What do you think?